edxadapt/edxadapt.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import pkg_resources
+
+from django.template import Template, Context
+from HTMLParser import HTMLParser
+from xblock.core import XBlock
+from xblock.fields import Scope, Boolean, Dict, List, String
+from xblock.fragment import Fragment
+from xblockutils.studio_editable import StudioEditableXBlockMixin
+
+from student.models import anonymous_id_for_user, User
+
+
+_ = lambda text: text
+log = logging.getLogger(__name__)
+html_parser = HTMLParser()
+
+
+class EdxAdaptXBlock(StudioEditableXBlockMixin, XBlock):
+    """
+    This XBlock automatically registers students in the EdxAdapt instance
+    using parameters configured by the teaching staff: params, skills, edx_adapt_api_url
+    """
+
+    has_author_view = True
+
+    display_name = String(
+        default='Adaptive Learning',
```

Comment:
maybe 'Adaptive Learning Excercise'?

that way if someone doesn't change the display name it makes more sense to learners?